scripts/lib/CIME/case.py

```diff
@@ -1076,7 +1077,8 @@ def get_mpirun_cmd(self, job="case.run"):
         mpi_attribs = {
             "compiler" : self.get_value("COMPILER"),
             "mpilib"   : self.get_value("MPILIB"),
-            "threaded" : self.get_build_threaded()
+            "threaded" : self.get_build_threaded(),
+            "unit_testing" : False
```

Comment:
This attribute shouldn't be required, putting it into the case.py may cause confusion?